core/src/main/java/tachyon/master/MasterInfo.java

```diff
@@ -1223,6 +1234,7 @@ public ClientDependencyInfo getClientDependencyInfo(int dependencyId)
   public ClientFileInfo getClientFileInfo(int fid) {
     synchronized (mRootLock) {
       Inode inode = mFileIdToInodes.get(fid);
+      mMasterSource.incGetFileStatusOps();
```

Comment:
@calvinjia , initially I wanted to record how many RPC calls happened for getting ClientFileInfo here. Yes, this makes it inconsistent with other inc*Ops. Do you think it is better to record how many attempts for all inc*Ops? If so, we simply call inc*Ops at the beginning of each methods and this will be more clear.